explorer/explorer.go

```diff
@@ -236,6 +253,10 @@ func New(dataSource explorerDataSourceLite, primaryDataSource explorerDataSource
 		"int64Comma": func(v int64) string {
 			return humanize.Comma(v)
 		},
+		"ticketWindowProgress": func(i int) float64 {
+			p := (float64(i) / 144) * 100
```

Comment:
144 needs to come from chaincfg.Params

Reply:
144 is also hardcoded in [root.tmpl](https://github.com/dcrdata/dcrdata/blob/a51c9e3d5914a13457aa0161e2f082a8d201c1b3/views/root.tmpl#L49) and in [extras.tmpl](https://github.com/dcrdata/dcrdata/blob/f865c40de6d833b6c5f461937058fe0826db5fda/views/extras.tmpl#L218). Somehow the `updateBlockData` js function needs to know the params, setting it as a data attribute in the html seems the simplest approach.

Reply:
Updated them in [extras.tmpl](https://github.com/RogueElement/dcrdata/blob/error-page/views/extras.tmpl#L218) and [home.tmpl](https://github.com/RogueElement/dcrdata/blob/error-page/views/home.tmpl#L47)